README.tfdoc.hcl

```diff
@@ -1059,9 +1059,28 @@ section {
 
             ```
             plaintext_secrets = {
-            SECRET_NAME_1 = "secret_value_1"
-            SECRET_NAME_2 = "secret_value_2"
-            ...
+              SECRET_NAME_1 = "secret_value_1"
+              SECRET_NAME_2 = "secret_value_2"
+            }
+            ```
+
+            When applied, a secret with the given key and value will be created in the repositories.
+            The value of the secrets must be given in plain text, github provider is in charge of encrypting it.
+            **Attention:** You might want to get secrets via a data source from a secure vault and not add them in plain text to your source files; so you do not commit plaintext secrets into the git repository managing your github account.
+          END
+        }
+
+        variable "encrypted_secrets" {
+          type        = map(string)
+          default     = {}
+          description = <<-END
+            This map allows you to create and manage encrypted secrets for repositories in your organization.
+            Each element in the map is considered a secret to be managed, being the key map the secret name and the value the corresponding secret in Base64 format.
+
+            ```
+            encrypted_secrets = {
```

Comment:
don't we have the `readme_example` property in terradoc to express that?

Reply:
yes! thanks for the reminder! :sweat_smile: